Sources/FoundationEssentials/UUID.swift

```diff
@@ -16,7 +16,7 @@ public typealias uuid_string_t = (Int8, Int8, Int8, Int8, Int8, Int8, Int8, Int8
 
 /// Represents UUID strings, which can be used to uniquely identify types, interfaces, and other items.
 @available(macOS 10.8, iOS 6.0, tvOS 9.0, watchOS 2.0, *)
-public struct UUID : Hashable, Equatable, CustomStringConvertible, Sendable {
+public struct UUID : Hashable, Equatable, CustomStringConvertible, Sendable, LosslessStringConvertible {
```

Comment:
This new conformance and the new `init` need to have the new availability annotation.